aws_xray_sdk/core/serverless_context.py

```diff
@@ -0,0 +1,131 @@
+import os
+import logging
+
+from .models.facade_segment import FacadeSegment
+from .models.segment import Segment
+from .models.mimic_segment import MimicSegment
+from .context import CXT_MISSING_STRATEGY_KEY
+from .lambda_launcher import LambdaContext
+from .context import Context
```

Comment:
This probably should not be used. The design is too complicated when you have 3 different context storage on a single module. The end goal is to make web app works on lambda. Feel free to refactor old classes.

Reply:
That's a fair point. I wanted to keep the "storage" portion of the context similar to its grandparent class, "Context" in case any changes were to be made with using ThreadLocal storage. Since it does make the design more complicated, I think explicitly using ThreadLocal instead of relying on the grandparent context to handle segment/subsegment storage would be better.